Cartfile.resolved

```diff
@@ -1,2 +1,2 @@
-github "ReactiveX/RxSwift" "3.0.1"
-github "daltoniam/Starscream" "2.0.2"
+github "ReactiveX/RxSwift" "3.4.0"
```

Comment:
I think it matters since it means that is what the project will resolve to when users do a `carthage bootstrap` , for example...